v2/spanner-common/terraform/samples/mysql-vpc-spanner/README.md

```diff
@@ -1,7 +1,7 @@
 ## Sample Scenario: MySQL in a VPC with Spanner
 
-> **_SCENARIO:_** This Terraform example illustrates launching a MySQL 5.7
-> in a GCE Compute instance inside a custom VPC subnet. It adds firewall
+> **_SCENARIO:_** This Terraform example illustrates launching multiple MySQL
```

Comment:
> i'm wondering in what scenario customers would launch MySQL instances as a part of the migration.

Customers are not only the intended audience. We have had multiple instances in the past where PSO/Partner/BB/SWEs have had to replicate a customer topology in order to understand and debug a customer use-case better. Every time, we have done a *best effort* job at helping these customers out, since we are unable to replicate.

This will empower the field and SWEs alike to perform higher fidelity replication and testing.

Reply:
I would still say this should be moved out into a different guide - which can be linked here. This seems to be a specific use case.
An alternative would be to just rename this guide to setup-mysql-and-migrate or something like that.

Reply:
I didn't get this, this is already a different guide, I think there is some confusion? Let me clarify. There are two guides in this PR -

1. A fresh guide written for configuring an existing VPC, which is in the `configure-shared-vpc` folder. It's purpose is to configure an existing VPC.
2. Changes to an existing guide which is in the `mysql-vpc-spanner` folder. We named this guide as `mysql-vpc-spanner` when this guide was [committed](https://github.com/GoogleCloudPlatform/DataflowTemplates/pull/1672), since it setups up MySQL inside a VPC + Spanner. The changes in this PR just modify that existing guide (whose purpose has always been test setup as per it's [README](https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/main/v2/spanner-common/terraform/samples/mysql-vpc-spanner/README.md)) to support sharded MySQL since single MySQL (original template) is a subset of the current one. Hence this version is more useful.

Hope that clarifies! Please let me know!